pandas/tests/util/test_util.py

```diff
@@ -417,6 +417,32 @@ def test_numpy_errstate_is_default():
     assert np.geterr() == expected
 
 
+@td.skip_if_windows
```

Comment:
Have these skips always been in place or is this a new addition in latest push?

Reply:
They've been there since I first added the tests.

Reply:
OK thanks. Can you move these into the `TestLocaleUtils` class? It seems like they should logically be grouped in there and that class already has the decorator to skip windows.

Should be the last change - thanks for hanging with the updates!

Reply:
OK, will do. Unfortunately I'll only be able to get to it tomorrow.